packages/plugin-e2e/playwright.config.ts

```diff
@@ -13,7 +13,7 @@ export default defineConfig<PluginOptions>({
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Retry on CI only */
-  retries: process.env.CI ? 2 : 0,
+  retries: process.env.CI ? 1 : 0,
```

Comment:
Is this change intended?

Reply:
oh, no it's wasn't supposed to be committed. thanks!